client/features/media/MediaDetail.jsx

```diff
@@ -1,43 +1,166 @@
 import React from 'react';
 import StarRatingInteractive from '../sharedComponents/StarRatingInteractive';
+import StarRating from '../sharedComponents/StarRating';
+import SearchBar from './../sharedComponents/SearchBar.jsx';
+import Reviews from './../reviews/Reviews.jsx';
+import test from './testData.js';
 import './MediaDetail.css';
 
-const MediaDetail = props => (
-  <div id="MovieDetail">
+//figure out how to wrap with error boundary and logger
+//need to find a way to get banner images to each movie/tv show
+
+class MediaDetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      mediaType: '',
+      name: '',
+      rating: '',
+      summary: '',
+      subscriptions: [],
+      reviews: [],
+      imgUrl: '',
+      userSubs: [],
+      watchWithSubscribed: [],
+      watchWithUnsubscribed: []
+    };
+    this.getMediaAndUserDetails = this.getMediaAndUserDetails.bind(this);
+    this.handleLogoClick = this.handleLogoClick.bind(this);
+  }
+
+  componentDidMount() {
+    this.getMediaAndUserDetails();
+  }
+
+  getMediaAndUserDetails() {
+    //this will be a request to the db to get media info to apply to state
```

Comment:
i dig all of the comments you laid throughout the file.  Seems like you probably started with the comments then built out the file =).  Thats the way to go 🚀 nice work